api/SVGGraphicsElement.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGGraphicsElement",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "30"
```

Comment:
http://mdn-bcd-collector.appspot.com/tests/api/SVGGraphicsElement doesn't have an instance because it's a base class for other things. It was introduced in https://chromium.googlesource.com/chromium/src/+/1c0121db5bdec1c757a78a9eb7b2888985809cd8 and probably moved some attributes around on the prototype chain. As a result the version numbers for subfeatures are nuanced. We may need `partial_implementation`.

Can you check if there are other interfaces updated in this PR for which we also don't test any instance? It might be good to break those out into its own PR. This PR is big, but until I got to this point seemed pretty straightforward.

Reply:
It looks like we don't have a custom test implemented for this API, oops.  I've reverted the changes to this and will add a custom test so we can test this properly.

Reply:
Great, thank you!

It was I who removed the custom test for `SVGComponentTransferFunctionElement` + `SVGGraphicsElement` + `SVGTextContentElement` + `SVGTextPositioningElement ` in https://github.com/foolip/mdn-bcd-collector/pull/643.

`SVGTextPositioningElement` isn't updated in this PR, but the remaining two are, and those tests also don't use any instances:
http://mdn-bcd-collector.appspot.com/tests/api/SVGComponentTransferFunctionElement
http://mdn-bcd-collector.appspot.com/tests/api/SVGTextContentElement